services/federator/test/unit/Test/Federator/InternalServer.hs

```diff
@@ -148,7 +148,7 @@ federatedRequestFailureAllowList =
         . runInputConst settings
         . runInputConst (FederationDomainConfigs AllowDynamic [FederationDomainConfig (Domain "hello.world") FullSearch FederationRestrictionAllowAll] 10)
         . interpretMetricsEmpty
-        $ callOutward targetDomain Brig (RPC "get-user-by-handle") request
+        $ callOutward Nothing targetDomain Brig (RPC "get-user-by-handle") request
```

Comment:
Kind of the same as above.

Reply:
Also not in scope, or I am missing something.